docs/general/ad-filtering/filter-policy.md

```diff
@@ -3,123 +3,289 @@ title: AdGuard filter policy
 
-**What shall these filters block?**
+- Anti-adblock that blocks website content without offering an alternative: to continue using the site, the user must disable the ad blocker on the site
+- “Anti-adblock wall” with alternative options offered, such as purchasing a subscription to continue using the ad blocker
+- Informational messages that do not obligate action: users are asked to disable the ad blocker, but these messages can be closed, allowing users to continue using the site without restrictions
+- Implementing alternative advertising in response to ad blocker usage: sites introduce alternative advertising when ad blockers are detected
+- Paywall: a method of content monetization where content is partially or fully hidden from users, with a message prompting them to purchase a subscription for access
 
-- These filters will block ads wherever possible.
-- Ads should be blocked regardless of their reasons and goals.
-- We will block ads caused by malicious apps or extensions that inject ads. Please note that we do it only on the condition that you specify how to install such an app or extension.
+### Filters
 
-**Limitations and Exceptions**
+AdGuard ad-blocking filters include:
 
-If a rule is subject to the list of limitations described below, then it won’t be added to the main filters.
+- AdGuard Base filter
+- AdGuard Mobile Ads filter
+- Regional filters divided by language principle — Chinese, Dutch, French, German, Japanese, Russian, Spanish/Portuguese, Turkish, and Ukrainian
 
-- Websites’ own ads should not be removed on purpose. On the other hand, they should not be unblocked if it may cause third-party ads to reappear.
-- Website-specific rules will be added only if a website has sufficient traffic. Traffic is determined by open statistics (if available) or by other means, such as followers on social media. A website's traffic is considered sufficient when it has 100 thousand visits per month. We will consider adding a rule for a website that's not that popular, but the final decision is up to the filter list maintainer.
-- Anti-adblock scripts will be blocked only if they limit or affect the functionality of a website.
-- Anti-adblock scripts will not be blocked in cases when it is prohibited by law.
-- Rules that often cause problems with the work of some websites will be deleted.
+### What these filters do
 
-## Tracking Protection filter
+- **Base filter** is designed to block ads on English-language websites and those for which there is no separate filter. It also contains general filtering rules that apply to all sites regardless of language.
+- **Mobile Ads filter** blocks advertisements on mobile versions of websites and within mobile applications. There is no segmentation based on language.
+- **Regional filters** follow the same policy as the **Base filter**, but limited to websites in certain languages.
```

Comment:
If there is a website that uses a language [^1] and an advert issue exists with only its mobile version, can the issue be addressed in the language-specific filters list?

[^1]: AdGuard has a filters list for the language.

Reply:
> If there is a website that uses a language 1 and an advert issue exists with only its mobile version, can the issue be addressed in the language-specific filters list?

Mobile filter

> I think that AdGuard Mobile Ads filters list is too huge for some users group that visits only some regional websites.
Therefore, I think that we need to optimize the filters list.

It has only 6.7k rules.

> I face some such complaints from South Korean users.

about what?

Reply:
> about what?

Reply:
> about what?

The fine delay during connecting a website.

Reply:
I doubt his/her diagnosis, even if it's true that will unlikely to be due to those regional rules. Such delay is usually caused by only one or a few probelmatic rules. The number of rules is not much releavant except for generic regex and cosmetic filters.